src/lvl_filesdk1.c

```diff
@@ -99,15 +99,11 @@ struct LegacyInitActionPoint { // sizeof = 8
     unsigned short num;
 };
 
-struct LegacyInitLight { // sizeof=0x14
```

Comment:
LegacyInitLight  is one that'll need to stay as is, as this is read from a binary file